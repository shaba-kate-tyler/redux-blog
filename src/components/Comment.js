import React from 'react';
import PropTypes from 'prop-types';

function Comment({ body, email }) {
  return (
    <>
      <p>{body}</p>
      <h4>-{email}</h4>
    </>
  );
}

Comment.propTypes = {
  body: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired
};

export default Comment;
