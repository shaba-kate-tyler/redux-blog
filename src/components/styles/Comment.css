.Comment {
    font-family: 'Roboto', sans-serif;
    padding: 10px;
    margin: 0;
    p {
        font-weight: 100
    }
}