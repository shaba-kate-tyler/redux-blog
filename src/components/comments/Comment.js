import React from 'react';
import PropTypes from 'prop-types';
import styles from '../styles/Comment.css';

function Comment({ body, email }) {
  return (
    <div className={styles.Comment}>
      <p>{body}</p>
      <h4>-{email}</h4>
    </div>
  );
}

Comment.propTypes = {
  body: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired
};

export default Comment;
