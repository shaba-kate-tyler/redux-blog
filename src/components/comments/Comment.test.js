import React from 'react';
import renderer from 'react-test-renderer';
import Comment from './Comment';

describe('Comment', () => {
  it('matches a snapshot', () => {
    const comment = {
      postId: 1,
      id: 1,
      name: 'id labore ex et quam laborum',
      email: 'Eliseo@gardner.biz',
      body: 'laudantium enim quasi est quidem magnam voluptate ipsam eos tempora quo necessitatibus dolor quam autem quasi reiciendis et nam sapiente accusantium'
    };

    const tree = renderer.create(
      <Comment {...comment}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
