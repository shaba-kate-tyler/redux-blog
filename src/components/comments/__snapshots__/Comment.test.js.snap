// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment matches a snapshot 1`] = `
<div
  className="Comment"
>
  <p>
    laudantium enim quasi est quidem magnam voluptate ipsam eos tempora quo necessitatibus dolor quam autem quasi reiciendis et nam sapiente accusantium
  </p>
  <h4>
    -
    Eliseo@gardner.biz
  </h4>
</div>
`;
